### ISSUE TITLE
No access to standard Node.js libraries like crypto

### ISSUE THREAD

**Issue by CQURE** [ngduyquockhanh/JScriptor#7]
Hello,
There is no access to standard node.js libraries like Crypto. 
For example when you try to run code:
require('crypto');

result is 
`<js>.:program(Unnamed:1)
org.graalvm.polyglot.Context.eval(Context.java:425)
org.JScriptor.HttpHandlerWithScript.handleHttpRequestToBeSent(HttpHandlerWithScript.java:79)
java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
java.base/java.lang.reflect.Method.invoke(Method.java:578)
burp.Zi4l.invoke(Unknown Source)
jdk.proxy2/jdk.proxy2.$Proxy28.handleHttpRequestToBeSent(Unknown Source)
burp.Ztee.Zl(Unknown Source)
burp.Zpzz.Zl(Unknown Source)
burp.Ztek.Zl(Unknown Source)
burp.Zvdk.ZK(Unknown Source)
burp.Zxbp.Zt(Unknown Source)
burp.Zxbp.ZI(Unknown Source)
burp.Zimg.ZI(Unknown Source)
burp.Zz8a.ZI(Unknown Source)
burp.Zie8.ZI(Unknown Source)
burp.Zj5z.ZO(Unknown Source)
burp.Zxiu.ZG(Unknown Source)
burp.Zxin.ZG(Unknown Source)
burp.Zcd_.ZM(Unknown Source)
burp.Zp7z.ZS(Unknown Source)
burp.Zcta.ZY(Unknown Source)
burp.Zdho.run(Unknown Source)
java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:577)
java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
java.base/java.lang.Thread.run(Thread.java:1623)
TypeError: Cannot load module: 'crypto'`

Maybe it is a case of Graalvm that does not support nodejs libraries in some cases?

Wish you good luck in solving this :) 

Cheers
Mike

**Comment by ngduyquockhanh** [comment]
That's right, graalvm currently does not support crypto. Use node-forge instead. I have plans to update to a new version that will allow the use of all nodejs modules
